templates/3dsmax/scripts/RepathRenderElements.ms

```diff
@@ -21,28 +21,24 @@ if(rem != undefined) then
 
 r = renderers.current     
 rendererName = r as string
-indexVrayAdv = findString rendererName "V_Ray_Adv_"
-indexVrayNext = findString rendererName "V_Ray_Next_"
+indexVRay5 = findString rendererName "V_Ray_5_"
 
-if (indexVrayAdv == 1 or indexVrayNext == 1) then
+if (indexVRay5 == 1) then
 (
+    tokens = filterString irAutoSavePath "\\"
```

Comment:
ìrAutoSavePath` is not defined here
`newPath` is used at two different places after but there should be two different values (one for `irAutoSavePath`, one for `irSavePath`)